src/virtual-machine.js

```diff
@@ -240,7 +240,7 @@ class VirtualMachine extends EventEmitter {
         const vm = this;
         const promise = storage.load(storage.AssetType.Project, id);
         promise.then(projectAsset => {
-            vm.loadProject(projectAsset.decodeText());
+            vm.loadProject(projectAsset.data.buffer);
```

Comment:
I would expect `projectAsset.decodeText()` to be the right thing here: `projectAsset.data` should always by a `Buffer` or `Uint8Array` regardless of the asset type, and in the case of a project it should be the project JSON encoded in UTF-8 form (see `Project` in https://github.com/LLK/scratch-storage/blob/develop/src/AssetType.js for example).

Are you saying that LLK/scratch-storage#32 makes it so that `projectAsset.data` now sometimes contains a ZIP file instead of JSON? If so... how is the storage module ending up with an exported project stored as `AssetType.Project`?